src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -183,23 +170,6 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
             RegisterEditorFactory(settingsEditorFactory);
         }
 
-        private async Task LoadOptionPersistersAsync(IComponentModel componentModel, CancellationToken cancellationToken)
```

Comment:
❓ Do we no longer need to force load these providers?

Reply:
I don't think it's needed.

Reply:
You sure? I admit I'm a bit worried here too.